packages/middleware/__tests__/unit/helpers/getAgnosticStatusCode.spec.ts

```diff
@@ -99,6 +99,17 @@ describe("[middleware-helpers] getAgnosticStatusCode", () => {
     expect(statusCode).toBe(expectedStatusCode);
   });
 
+  it("retrieves status code for axios even when status code is missing", () => {
+    const testData = {
+      isAxiosError: true,
+      response: {},
```

Comment:
```suggestion
```
Let's remove the response (as we don't have it in timed out response) to make sure it doesn't throw if it's missing.